services/web/client/source/class/osparc/desktop/ControlsBar.js

```diff
@@ -50,6 +50,13 @@ qx.Class.define("osparc.desktop.ControlsBar", {
   members: {
     __startButton: null,
     __stopButton: null,
+    __groupButton: null,
+    __ungroupButton: null,
+
+    setIsWorkbenchVisible: function(isWorkbenchContext) {
```

Comment:
minor: naming conventions ``is`` is only used for boolean getter method. set+is sounds weird... maybe call it setWorkbenchVisibility and directly pass it the string value, or pass it a boolean and do the logic there, that's fine